README.md

```diff
@@ -930,8 +928,10 @@ Use a different icon set for all flash messages:
 ```php
 $this->loadHelper('Html', [
     'className' => 'BootstrapUI.Html',
-    'iconSet' => 'fas',
-    'prefix' => 'fa',
+    'iconDefaults' => [
+        'iconSet' => 'fas',
```

Comment:
should this be then just "set" or "namespace"?
As the prefix isn't needed anymore with the nesting here?

Reply:
I've just documented this wrongly initially, `iconDefaults` was always required. Certainly the option could be renamed, but the `prefix` option is still required - or did you mean something different by "prefix"?

Reply:
icon prefix i meant. and yes, prefix also doesn't have a prefix :)
So for consistency I wonder if both should be
- set/namespace
- prefix

only

Reply:
In my icon helper I also allowed prefix to be bool true btw, to auto use the namespace (like fa and fa-icon)
But this is maybe not needed here or out of scope - just wanted to give that input here.

Reply:
Yeah, let's keep the additional functionality for a new PR at this point, I have some more ideas too, like making the set/namespace optional, using a template based solution for easier SVG sprite usage, etc :)

`namespace` sounds good to me.